spec/features/user_can_sign_in_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.feature "Session/ Sign-in", type: :feature do
+
+  before(:each) do
+    successful_sign_up
+  end
+
+  scenario "A user can sign in with valid credentials" do
+
+    visit "/session/new"
+    fill_in :email, with: "elishka@keepingitrails.com"
+    fill_in :password, with: "pa55w0rd"
+    click_button "Log In"
+
```

Comment:
can we take out the blank lines in the feature tests just for neatness?

Reply:
👍